src/Microsoft.DotNet.XHarness.CLI/CommandArguments/Argument.cs

```diff
@@ -158,7 +157,7 @@ public override void Action(string argumentValue)
     public abstract class StringArgument : Argument<string?>
     {
         public StringArgument(string prototype, string description)
-            : base(prototype, description)
+            : base(prototype, description, null)
```

Comment:
is it ok to have no opportunity to pass default value to string argument?

Reply:
It's a `string?` so I think this is expected

Reply:
There's also a `RequiredStringArgument` which has `string` only